src/components/list-archive-all-tags/list-archive-all-tags.js

```diff
@@ -0,0 +1,60 @@
+import * as React from "react"
+
+import "../../scss/styles.scss"
+import "../../js/globals"
+
+import ListItem from "../list-item/list-item"
+import Chip from "../chip/chip"
+import { useStaticQuery, graphql } from "gatsby"
+
+const ListArchiveAllTags = ({
+  background,
+  noSpace,
+}) => {
+
+  const data = useStaticQuery(graphql`
+    query {
+      allContent {
+        group(field: components___hero___kangaroo___tags) {
+          fieldValue
+          totalCount
+        }
+      }
+    }
+  `)
+
+  let container = "container-xxl"
+  let row = "row"
```

Comment:
We shouldn't define these as variables

Reply:
It's a "copy" of other code we use in many components. We have to change all later accordingly IMO for consistency, not just here.